updateRepo.sh

```diff
@@ -1,34 +1,53 @@
+#!/bin/bash
+
 echo "Checking if repo needs to be updated"
 export ORG="IBM-Swift"
 export REPO="generator-swiftserver-projects"
 export GH_REPO="github.com/${ORG}/${REPO}.git"
-export BRANCH="init"
+export BRANCHES="init openAPI"
```

Comment:
Why do all these need to be exported?

Reply:
I don't think they do, I'm not 100% sure why they were exported in the first place but I can't see that they're being used elsewhere.

Reply:
OK cool, let's fix this script up properly then :)